Fixed controller Input

thecrazy opened the pull request:
-Added the missing input check required to save while using a controller
-Suppressed the input to prevent the menu from closing as soon as it opens

thecrazy committed:
Fixed controller Input

-Added the missing input check
-Suppressed the input to prevent the menu from closing as soon as it opens

DelphinWave reviewed (changes requested):
Hi, the changes look good.

Is it necessary to add Harmony though? I think that can cause compatibility issues and it doesn't seem to be used in your code.

Have you tested these changes locally? If it works for you locally and you remove Harmony (unless it's needed) then it's probably good to go.

thecrazy commented:
Mmm... strange. I installed it cause on compiled it complained about it missing.

I will take a look tonight.

thecrazy commented:
Still not sure what happened, but I removed Harmony, and it works fine without it.

I made other changes since the last pull request.

- I modified the way horse sprites are loaded. You don't need to rename the files and set the count in the config anymore. All PNG files in the asset folders will be loaded automatically.
- I also modified the menu, so it displays your current horse when opening.

I tested it in single, split screen and multiplayer.

Still got some cleaning up to do before submitting it, but it's coming your way soon.

DelphinWave commented:
> 
> 
> Still not sure what happened, but I removed Harmony, and it works fine without it.
> 
> I made other changes since the last pull request.
> 
>     * I modified the way horse sprites are loaded. You don't need to rename the files and set the count in the config anymore. All PNG files in the asset folders will be loaded automatically.
> 
>     * I also modified the menu, so it displays your current horse when opening.
> 
> 
> I tested it in single, split screen and multiplayer.
> 
> Still got some cleaning up to do before submitting it, but it's coming your way soon.

Hi again. I might have time to get working on my mods again during the week of Oct 18th.
Did you end up sending along those updates you mentioned? I don't see them on my end, but I'm also a bit new to the pull request flow.

In any case, if you end up working on it again, I probably won't get to it until the 18th. That week it would be nice to implement your changes and update the mod on Nexus though :)
